server/mailer/templates/registrationTemplate.ejs

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+  </head>
+  <body>
+    <div>
+      <div
+        style="
+          font-weight: 300;
+          font-size: 1.1em;
+          color: rgba(0, 0, 0, 0.4);
+          margin-top: 10px;
+        "
+      >
+        <h1><%= name %> using <%= mail %></h1>
+        <h2>is requesting access to dpdash</h2>
+        <a
+          style="text-decoration: none"
+          href="<%= process.env.HOME_URL %>"
```

Comment:
let's pass this in to the template like we do the user so we're not mixing where the data is coming from.

Reply:
[d7d81f3](https://github.com/AMP-SCZ/dpdash/pull/457/commits/d7d81f3f15d9919a2e74cb19a78f4a87d984c42c)